frontend/www/js/omegaup/api.js

```diff
@@ -614,7 +615,18 @@ export default {
 
     changePassword: _call('/api/user/changepassword/'),
 
-    contestStats: _call('/api/user/conteststats/'),
+    contestStats: _call('/api/user/conteststats/', function(data) {
+      let contests = [];
+      for (let contestAlias in data.contests) {
+        const now = new Date();
+        const currentTimestamp = data.contests[contestAlias].finish_time * 1000;
```

Comment:
nit:

```suggestion
        const currentTimestamp = data.contests[contestAlias].data.finish_time * 1000;
```

(esto acaba de cambiar en #3117).